app/checklist/views.py

```diff
@@ -23,6 +25,27 @@ def index():
     default_checklist_items = DefaultChecklistItem.query.all()
     return render_template('checklist/index.html', default_checklist_items=default_checklist_items)
 
+@checklist.route('/upload', methods=["GET", "POST"])
+def upload():
+    """Upload a pdf."""
+    form = UploadDocumentForm()
+
+    doc = Document.query.get(current_user.id)
```

Comment:
If `doc` is None, we get an error later in line 45 when we call `form.file_urls.data = doc.document_urls`.